EIPS/eip-3074.md

```diff
@@ -0,0 +1,159 @@
+
+## Abstract
+
+This EIP creates two precompiles:
+ - `CALL_PRECOMPILE` - forwards a `CALL`, setting `CALLER` according to an ECDSA signature, using a relayer-sponsee nonce for replay protection.
+ - `NONCE_PRECOMPILE` - provides access to relayer-sponsee nonces expected by `CALL_PRECOMPILE`.
+
+## Motivation
+
+Sponsored transactions—the separation of fee payment from transaction content—have been a long standing feature request. Unlike similar proposals, this EIP specifies a method of implementing sponsored transactions that allows both externally owned accounts (EOAs) and [EIP-2938](./eip-2938.md) contracts to act as sponsors.
+
+With the explosion of tokens built on Ethereum, especially stable coins, it has become common for EOAs to hold valuable assets without holding any Ether at all. These assets must be converted to Ether before they can be used to pay gas fees, but without Ether to pay for the conversion, it's impossible to convert them. Sponsored transactions break the circular dependency.
+
+While it is possible to emulate sponsored transactions (ex. [Gas Station Network](https://www.opengsn.org/)), these solutions require specific support in callee contracts.
+
+## Specification
+
+Sponsored transactions are implemented with the addition of two precompiles:
+
+ - The first, at address `0x13`, which functions like a `CALL` instruction that additionally sets the caller address based on an ECDSA signature.
+ - The second, at address `0x14`, provides access to the current nonce for the given relayer-sponsee pair.
+
+### Definitions
+
+ - **`CALL_PRECOMPILE`** - the specific precompile at address `0x13`, introduced by this EIP, which implements the `CALL` analogue.
+ - **`NONCE_PRECOMPILE`** - the specific precompile at address `0x14`, introduced by this EIP, which exposes relayer-sponsee nonces.
+ - **Transaction-like Package** - the signed arguments passed to `CALL_PRECOMPILE`.
+ - **Sponsor** - the account which is responsible for paying gas fees and sending the transaction. May or may not be the same as the relayer.
+ - **Sponsee** - the account which signed the transactions-like package.
+ - **Relayer** - the account or contract which directly calls into `CALL_PRECOMPILE`. May or may not be the same as the sponsor.
+ - **Callee** - the target of the call from `CALL_PRECOMPILE`.
+
+### API
+
+#### `CALL_PRECOMPILE`
+
+`CALL_PRECOMPILE` requires the following eight arguments:
+
+ - `nonce` - the next nonce value, as described below;
+ - `to` - the address of the callee (not `CALL_PRECOMPILE`);
+ - `gaslimit` - the minimum gas limit which must be provided with the call into `CALL_PRECOMPILE`;
+ - `value` - the exact amount of Ether in wei to transfer from the relayer to the callee;
+ - `data` - the calldata for the call into `to`; and
+ - `v`, `r`, `s` - signature for the package, including chain id as specified in [EIP-115](./eip-115.md).
```

Comment:
```suggestion
 - `v`, `r`, `s` - signature for the package, including chain id as specified in [EIP-155](./eip-155.md).
```
I'm pretty sure this is what you meant.  Thank God for the EIP validator that caught this (it is what is blocking CI from letting me merge)!